app/main/views/questionnaire.py

```diff
@@ -30,8 +30,13 @@
                                     import_name=__name__,
                                     url_prefix='/questionnaire/<eq_id>/<form_type>/<collection_id>/')
 
+action_blueprint = Blueprint(name='action',
```

Comment:
Not sure if we need the blueprint at all. There's been some discussion about having a single endpoint to avoid using HTML5 formaction attribute in the template. If I re-work this so that it posts back to the same endpoint then we may just be able to use the questionnaire endpoint or some variation on it.